middleware/kubernetes/k8sclient/k8sclient.go

```diff
@@ -91,8 +118,9 @@ func (c *K8sConnector) GetServicesByNamespace() map[string][]ServiceItem {
 }
 
 
+// GetServiceItemInNamespace returns the ServiceItem that matches
+// servicename in the namespace
```

Comment:
Called from line 105 in middleware/kubernetes/kubernetes.go

Mostly this is to abstract out the search logic to be separate from the logic in kubernetes.Records().